bin/json_schemas/complete_config_schema.json

```diff
@@ -676,9 +683,10 @@
             {
               "type" : "array",
               "items" : {
-                "type" : "string"
+                "type" : "string",
+                "minLength": 1
```

Comment:
The `minLength` is included here as the field above this (which is part of the containing `anyOf` is a string with `minLength: 1`.